### DISCUSSION TITLE
[help]网站生成的sitemap路径不对

### DISCUSSION THREAD

**Discussion by lkzwc** [withastro/docs#7839] [Ideas]
网站生成的sitemap不对 不知道是哪里的问题 有大佬帮忙看看嘛

比如我的网站下边有分类 blog 、post

生成的sitemap 路径是 www.domain.cn/blog//xxx  www.domain.cn/post//xxx 多了一个/ 怎么回事

**Comment by liruifengv** [comment]
你好，可以给一个最小复现demo么

**Reply by lkzwc** [reply]
https://github.com/lkzwc/liuketh/tree/e0e4bd492dbbb22d59e46d8ba7202e1f038c89af  大佬 这个仓库 你帮忙看下

**Reply by lkzwc** [reply]
感觉在内容集合上有出的问题 e0e4bd492dbbb22d59e46d8ba7202e1f038c89af  最开始出问题的地方

**Reply by liruifengv** [reply]
你的内容集合的slug写的是`slug: /9zhe`，就会多出来一个 `/`。默认slug会是文件名，不用指定，指定的话前面不要加/
